dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/css/OpenAPIUIBundlesUpdater.java

```diff
@@ -295,7 +284,12 @@ private static void writeStreamToZip(InputStream stream, ZipOutputStream zos) th
 
     private static boolean waitForBundlesToStart(Set<Bundle> openAPIUIBundles) {
         try {
-            new OpenAPIUIBundlesListener(openAPIUIBundles).await();
+            BundleContext bundleContext = FrameworkUtil.getBundle(OpenAPIUIBundlesUpdater.class).getBundleContext();
+            if(bundleContext != null){
+                new OpenAPIUIBundlesListener(openAPIUIBundles, bundleContext).await();
+            } else {
+                return false;
+            }
```

Comment:
We might want a comment here to say why the bundleContext may be null and why it's safe to ignore it and return `false`.